src/Core/src/Sts/Input/AssumeRoleRequest.php

```diff
@@ -163,9 +163,9 @@ public function getPolicy(): ?string
     }
 
     /**
-     * @return PolicyDescriptorType[]
+     * @return PolicyDescriptorType[]|null
      */
-    public function getPolicyArns(): array
+    public function getPolicyArns(): ?array
```

Comment:
sadly that's a BC break :(

Reply:
Yes for now, I don't implement the first solution to avoid BC break. But it stills possible